src/app/gateways/components/RuleMatch.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <dl class="match-definition-list">
+    <template v-if="props.match.method">
+      <div class="match">
+        <dt>
+          <KBadge appearance="neutral">
+            Method
+          </KBadge>
+        </dt>
+
+        <dd>
+          <KBadge>{{ props.match.method }}</KBadge>
+        </dd>
+      </div>
+    </template>
+
+    <template v-if="props.match.path">
+      <div class="match">
+        <dt>
+          <KBadge appearance="neutral">
+            Path
+          </KBadge>
+        </dt>
+
+        <dd>
+          <span class="key">Type:</span> <span>{{ props.match.path.type }}</span>
+          <span class="key">Value:</span> <code>{{ props.match.path.value }}</code>
+        </dd>
+      </div>
+    </template>
+
+    <template v-if="props.match.queryParams && props.match.queryParams.length > 0">
+      <div
+        v-for="(param, index) in props.match.queryParams"
+        :key="index"
+      >
+        <div class="match">
+          <dt>
+            <KBadge appearance="neutral">
+              Query parameter
+            </KBadge>
+          </dt>
+
+          <dd>
+            <span class="key">Type:</span> <span>{{ param.type }}</span>
+            <span class="key">Name:</span> <span>{{ param.name }}</span>
+            <span class="key">Value:</span> <span>{{ param.value }}</span>
```

Comment:
Tiny thing: What do you think of a nested `dl` here?

Reply:
I briefly thought about it, and didn’t really have a clear picture about it. It feels a bit "spammy" to mark up everything as a definition list. I’m not sure how meaningful the notion of "this term is defined by another definition list" is.